mne/viz/tests/test_topomap.py

```diff
@@ -557,6 +557,19 @@ def test_plot_topomap_bads():
     plt.close('all')
 
 
+def test_plot_topomap_bads_grad():
+    """Test plotting topomap with bad gradiometer channels (gh-8802)."""
+    import matplotlib.pyplot as plt
+    data = np.random.RandomState(0).randn(203)
+    info = read_info(evoked_fname)
+    info['bads'] = ['MEG 2242']
+    picks = pick_types(info, meg='grad')
+    info = pick_info(info, picks)
+    assert len(info['chs']) == 203
+    plot_topomap(data, info, res=8)
+    plt.close('all')
```

Comment:
Same here, no need to `plt.close('all')` at the end, it happens automatically via pytest now